scripts/BUILD

```diff
@@ -66,13 +54,13 @@ set -eufo pipefail
 
 cwd="\\$${BUILD_WORKSPACE_DIRECTORY:=\\$$(pwd)}"
 javaformat="\\$$1"
-buildifier="\\$$2"
 
 # Format java code
 find "\\$$cwd" -type f -name "*.java" |  xargs "\\$${javaformat}" --replace
 
 # Run buildifier
-\\$${buildifier} --lint fix --mode fix -r "\\$$cwd"
+cd \\$$cwd
+bazel run @buildifier_prebuilt//buildifier -- --lint fix --mode fix -r "\\$$cwd"
```

Comment:
Does using buildifier as a data dep not work? `bazel` running `bazel run` is a little odd.